floyd/client/base.py

```diff
@@ -4,6 +4,13 @@
 import floyd
 from floyd.manager.auth_config import AuthConfigManager
 from floyd.exceptions import AuthenticationException, BadRequestException, NotFoundException, OverLimitException
+
+from floyd.exceptions import AuthenticationException, AuthorizationException, \
```

Comment:
Seems like some of these are already imported above.

Also, can you use [parantheses](https://www.python.org/dev/peps/pep-0328/#rationale-for-parentheses) for grouping these imports? That is the pattern we use in the rest of the code.